src/core/lib/transition-manager.js

```diff
@@ -195,7 +195,7 @@ export default class TransitionManager {
     this.state.viewport = extractViewportFrom(Object.assign({}, this.props, viewport));
 
     if (this.props.onViewportChange) {
-      this.props.onViewportChange(this.state.viewport);
+      this.props.onViewportChange(this.state.viewport, {inTransition: true});
```

Comment:
Approved for now, but this is an API change that will need to be documented, so please add a note to API audit list.
Ultimately, I think we may want to add a new callback `onViewStateChanged` which takes a `ViewState` and a controller id, in this case we can add these things.

Reply:
Added to API Audit list.